### PR TITLE
Add WebSocket authentication options.

### LINKED ISSUE #1728: Document support for enabling and configuring TLS/mTLS in WebSocket service.
Document the following CLI options:
To set keystore path, password and type for TLS
To set trust store path, password and type for mTLS

For more information, see https://github.com/hyperledger/besu/pull/7854

### PR DESCRIPTION
**Pull request by bgravenorst**
## Description
Document the WebSocket SSL/TLS authentication options.

- 

### Issue(s) fixed
Fixes #1728 

### Preview

https://besu-docs-git-fork-bgravenorst-doc-1728-hyperledger.vercel.app/public-networks/reference/cli/options#rpc-ws-ssl-enabled

### CODE REVIEW

**Review by fab-10 — APPROVED** [review]

File: docs/public-networks/reference/cli/options.md [hunk 1]
```diff
@@ -4884,6 +4887,489 @@ rpc-ws-port="6174"
...
+Enables or disables client authentication for the WebSocket JSON-RPC service. The default is `false`.
+
+:::note
+
+When enabling client authentication, specify the truststore type using [`--rpc-ws-ssl-truststore-type`](#rpc-ws-ssl-truststore-type)
+and provide the appropriate file path for the truststore or trust certificate using either
+[`--rpc-ws-ssl-truststore-file`](#rpc-ws-ssl-truststore-file) (for JKS or PKCS12) or
+[`--rpc-ws-ssl-trustcert-file`](#rpc-ws-ssl-trustcert-file) (for PEM).
+
+If using JKS or PKCS12, you also need to specify the truststore password with [`--rpc-ws-ssl-truststore-password`](#rpc-ws-ssl-truststore-password).
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:4983]:
```suggestion
If using JKS or PKCS12, specify the truststore password using [`--rpc-ws-ssl-truststore-password`](#rpc-ws-ssl-truststore-password).
```

File: docs/public-networks/reference/cli/options.md [hunk 2]
```diff
@@ -4884,6 +4887,489 @@ rpc-ws-port="6174"
...
+```
+
+</TabItem>
+
+</Tabs>
+
+Enables or disables server SSL/TLS authentication for the WebSocket JSON-RPC service. The default is `false`.
+
+Set the appropriate keystore type with the [`--rpc-ws-ssl-keystore-type`](#rpc-ws-ssl-keystore-type)
+command line option.
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:5027]:
```suggestion
Set the appropriate keystore type using [`--rpc-ws-ssl-keystore-type`](#rpc-ws-ssl-keystore-type).
```

File: docs/public-networks/reference/cli/options.md [hunk 3]
```diff
@@ -4884,6 +4887,489 @@ rpc-ws-port="6174"
...
+rpc-ws-ssl-key-file="/home/me/me_node/websocket-cert.pem"
+```
+
+</TabItem>
+
+</Tabs>
+
+Path to the PEM certificate file when enabling SSL/TLS for the WebSocket JSON-RPC service.
+This file contains the private key that corresponds to the public certificate specified using
+[`rpc-ws-ssl-cert-file`](#rpc-ws-ssl-cert-file).
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:5069]:
```suggestion
[`--rpc-ws-ssl-cert-file`](#rpc-ws-ssl-cert-file).
```

File: docs/public-networks/reference/cli/options.md [hunk 4]
```diff
@@ -4884,6 +4887,489 @@ rpc-ws-port="6174"
...
+rpc-ws-ssl-keystore-file="/home/me/me_node/keystore.jks"
+```
+
+</TabItem>
+
+</Tabs>
+
+Path to the keystore file when enabling SSL/TLS for the WebSocket JSON-RPC service.
+The keystore file is used to store the server's private key and public certificate in a single
+file, typically in JKS or PKCS12 format. This option is used when you prefer to
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:5113]:
```suggestion
file, typically in JKS or PKCS12 format. Use this option if you prefer to
```

File: docs/public-networks/reference/cli/options.md [hunk 5]
```diff
@@ -4884,6 +4887,489 @@ rpc-ws-port="6174"
...
+
+</TabItem>
+
+</Tabs>
+
+Path to the truststore file for enabling SSL/TLS client authentication for the WebSocket JSON-RPC
+service.
+
+Specify the truststore file password with the [`rpc-ws-ssl-truststore-password`](#rpc-ws-ssl-truststore-password)
+command.
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:5286]:
```suggestion
Specify the truststore file password using [`--rpc-ws-ssl-truststore-password`](#rpc-ws-ssl-truststore-password).
```

**Review by alexandratran — APPROVED** [review]
Minor suggestions for consistency.

**Pull request merged**